gitlab/gitlab_admin.sh

```diff
@@ -227,6 +227,41 @@ create_api_token() {
   echo "${token}"
 }
 
+check_api_token_validity() {
+  local username="${1:-}"
+  _check_parameter "username" "${username}"
+  local user_id
+  user_id="$(_get_id_from_username "${username}")"
+  local name="CI token"
+
+  impersonation_tokens=$(curl -sSL --header "${TOKEN_HEADER}" \
```

Comment:
Expired token may be removed from gitlab and therefore return something like: 
{"error":"invalid_token","error_description":"Token is expired. You can either do re-authorization or token refresh."}

Reply:
This method is used as well for logging expired date of tokens.

NOTE: expired tokens are not visible from the UI, but the API returns a full list.